src/fstests/runner.py

```diff
@@ -30,7 +29,19 @@ def __init__(self, configs, args):
         runtime_config = configs['labs']['shell']
         self._runtime = kernelci.lab.get_api(runtime_config)
 
-    def _schedule_job(self, node, device_config, tmp):
+    def _create_node(self, tarball_node, plan_config):
+        print(tarball_node)
+        node = {
+            'parent': tarball_node['_id'],
+            'name': plan_config.name,
+            'artifacts': tarball_node['artifacts'],
+            'revision': tarball_node['revision'],
+            'path': tarball_node['path'],
```

Comment:
Not quite, it needs to be the parent node and the current node.  See `src/runner.py`:
```python
'path': tarball_node['path'] + [plan_config.name],
```

Reply:
ok, now I see. It was in the staging branch.